### ISSUE TITLE
Mismatch of learning rate with the selfcalib paper and config

### ISSUE THREAD

**Issue by Theo-Cheynel** [TRI-ML/vidar#10]
Hi, I notices the [Self-supervised self-calibration paper](https://arxiv.org/pdf/2112.03325.pdf) mentions, on the fourth page, a 1e-3 learning rate for the network that regresses the intrinsics params of the camera, however the code uses 1e-2 instead :https://github.com/TRI-ML/vidar/blob/a306e23cc47ae05a0c35b0bf8acf7112c87c049c/configs/papers/selfcalib/ucm_kitti.yaml#L35 
Is this a mistake ? Which one is correct ?

Edit : same thing for the batch size, the paper uses 16 but the config file uses 8

**Comment by kudo1026** [comment]
Thanks for pointing it out. The paper used a slightly different code base, and the actual optimal combination of learning rate and batch size may depend on your machine. But for the most part, you can go with the config provided with this code base, and you are welcome to tweak around it for your own dataset.

**Comment by Theo-Cheynel** [comment]
Awesome, thanks ! Currently training it to see if I can replicate the performance before training it on our datasets, since I'm not familiar at all with the codebase :smile:

**Comment by KOuldamer** [comment]
Hello, 
I trained self calibration network on euroc dataset using **ucm_euroc.yaml** config. However, I don't know how to use the resulted model to estimate the intrinsics parameters. Could you help me find the inference configuration for self. calibration please.

**Comment by MichaelWangGo** [comment]
> Hello, I trained self calibration network on euroc dataset using **ucm_euroc.yaml** config. However, I don't know how to use the resulted model to estimate the intrinsics parameters. Could you help me find the inference configuration for self. calibration please.

Hi, I really want to know how to run a .yaml file for a deep learning network. I think there should be a main.py or something like that. Appreciated it if you can tell me how to train the network.
